### PR TITLE
Support/wagtail 50

### PR DESCRIPTION
**Pull request by lparsons396**
- Update testing matrix to include Wagtail 5.1
- Drop support for Python 3.7

### CODE REVIEW

File: CHANGES [hunk 1]
```diff
@@ -1,5 +1,7 @@
 Unreleased
 ==========
+- Update for Wagtail 5.1
+- Drop support for Django 3.7
```
Comment (zerolab) [inline CHANGES:4]: https://github.com/wagtail/wagtail-factories/blob/main/setup.py#L57 needs removing

File: tox.ini [hunk 2]
```diff
@@ -1,9 +1,9 @@
 [tox]
 minversion = 3.14.6
 envlist =
-    py{37,38,39,310}-django32-wagtail{41,42,50}-factoryboy32
-    py{38,39,310}-django40-wagtail{41,42,50}-factoryboy32
-    py{38,39,310,311}-django41-wagtail{41,42,50}-factoryboy32
+    py{38,39,310}-django{32,41}-wagtail{41,42,50,51}-factoryboy32
```
Comment (zerolab) [inline tox.ini:4]:
```suggestion
    py{38,39,310}-django{32,41}-wagtail{41,50,51}-factoryboy32
```

File: tox.ini [hunk 3]
```diff
@@ -1,9 +1,9 @@
...
+    py311-django41-wagtail{41,42,50,51}-factoryboy32
```
Comment (zerolab) [inline tox.ini:5]:
```suggestion
    py311-django41-wagtail{41,50,51}-factoryboy32
```

Comment (zerolab) [reply tox.ini:5]: and the `wagtail42: wagtail>=4.2,<5.0` line can go

**Review by zerolab — CHANGES_REQUESTED** [review]

**Comment by katdom13** [comment]
Superseded by: https://github.com/wagtail/wagtail-factories/pull/81

**Pull request closed without merging**